### ISSUE TITLE
Please visualise replications resulting from change of Storage Class

### ISSUE THREAD

**Issue by onlyjob** [moosefs/moosefs#365]
When a _storage class_ definition is changed, replication begins but there is no indication of how many chunks are out of place in the "CGI/Resources" page.

It would be very useful to add some information to get the idea of the numbers of chunks that are due to be relocated.

Thanks.

(Branched off https://github.com/moosefs/moosefs/issues/249#issuecomment-604324484)

**Comment by chogata** [comment]
We cannot do it on-line, this would take too much resources. We could make statistics with every chunk loop completed by the master and store information for each storage class: how many chunks were on "wrong labels" for that storage class, how many were replicated, how many still need to be replicated and display that info. Do you think that would be useful?

**Comment by onlyjob** [comment]
Yes that would be useful. Thanks. I had replication running for over two weeks and statistics generated per chunk loop is definitely useful in a long run.

**Comment by chogata** [comment]
Ok, I'm adding it to the feature list then.
